### PR TITLE
Add model cache refreshes monitoring page

### PR DESCRIPTION
**Pull request by kulyk**
Adds a model cache refreshes monitoring page to "Admin > Tools". Tools used to be available only in paid Metabase plans, but model persistence monitoring should be available in OSS too. So this PR also changes how Metabase includes the "Tools" tab. By default, it'd always be shown with the monitoring data. In paid plans, on the other hand, it will have a tab bar where people can change between "Erroring Questions" and "Model cache log" pages

### Demo

CleanShot 2022-04-08 at 19 20 49@2x

CleanShot 2022-04-08 at 19 21 28@2x

### CODE REVIEW

**Comment by alxnddr** [comment]
@kulyk we recently introduced Monitoring global permission that you can enable [here](http://localhost:3000/admin/permissions/general). It gives access to "Tools" navigation item for non-admins and they should be able to perform all actions but without data leaks. For instance, from "Questions that errored when last run" tab users will be able to rerun queries without receiving actual data. I think you will need something like this too

**Comment by kulyk** [comment]
> @kulyk we recently introduced Monitoring global permission that you can enable [here](http://localhost:3000/admin/permissions/general). It gives access to "Tools" navigation item for non-admins and they should be able to perform all actions but without data leaks. For instance, from "Questions that errored when last run" tab users will be able to rerun queries without receiving actual data. I think you will need something like this too.
> 
> 
> 
> For some reason on this branch I could not access admin settings as a non-admin user

@alxnddr was it merged just recently or some time ago? I suspect the feature branch can be a bit outdated

**Comment by alxnddr** [comment]
> @alxnddr was it merged just recently or some time ago? I suspect the feature branch can be a bit outdated

@kulyk the branch is pretty outdated but it contains the monitoring permission code. I'd suggest to rebase on the latest master and check if it fixes the issue

File: enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorOverview.jsx [hunk 1]
```diff
@@ -69,46 +69,49 @@ export default function ErrorOverview(props) {
...
-            collectionFilter,
-            sorting.column,
-            getSortOrder(sorting.isAscending),
-          )}
-          className={"mt2 bounded-overflow-x-scroll"}
-        />
-      )}
-    </AuditParameters>
+    <>
+      <h2>Questions that errored when last run</h2>
```
Comment (alxnddr) [inline enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorOverview.jsx:73]: Missing translation

File: frontend/src/metabase/admin/routes.jsx [hunk 2]
```diff
@@ -170,6 +178,20 @@ const getRoutes = (store, CanAccessSettings, IsAdmin) => (
         {getAdminPermissionsRoutes(store)}
       </Route>
 
+      <Route path="tools" component={IsAdmin}>
```
Comment (alxnddr) [inline frontend/src/metabase/admin/routes.jsx:181]: Please replace `IsAdmin` with createAdminRouteGuard("tools") to allow non-admin users with the monitoring permission open the route

**Review by alxnddr — APPROVED** [review]
@kulyk I found what was the issue, please take a look at my comment and fix before merging.
Code looks good, however I could not test the functionality because even after a while with enabled model persistence nothing appears on the page

File: enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorOverview.jsx (hunk 1 above)
Comment (kulyk) [reply enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorOverview.jsx:73]: Thanks! Will fix

File: frontend/src/metabase/admin/routes.jsx (hunk 2 above)
Comment (kulyk) [reply frontend/src/metabase/admin/routes.jsx:181]: Oh, got it, thanks for investigating that!

**Pull request merged**